layers/meta-opentrons/recipes-robot/opentrons-usb-bridge/files/opentrons-usb-bridge.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Opentrons USB Bridge
+Requires=nginx.service
+After=nginx.service
+
+[Service]
+Type=notify
+ExecStart=python3 -m ot3usb
+StateDirectory=opentrons-usb-bridge
+Environment=PYTHONPATH=/opt/:/usr/lib/python3.8/site-packages
```

Comment:
This wants to be `/opt/ot3usb` i believe if you're putting a bundle in that directory

Reply:
the `pipenv_app_bundle` machinery puts the specified wheel and all its deps in the specified directory, so that specified directory looks like a virtualenv's lib/site-packages. So you need to specify the full directory for the python path, and a proper install of the `ot3usb` package will be inside